backend/open_webui/retrieval/vector/dbs/milvus.py

```diff
@@ -1,9 +1,9 @@
+import os # Added import
```

Comment:
After your latest changes, the import os, is not required any more, since you are importing variables from config.